website/docs/core-concepts/connecting-to-data-sources/authentication/README.md

```diff
@@ -45,11 +45,11 @@ When you want to ensure that the incoming requests originate from Appsmith, you
 
 ### Authentication Type
 
-You can define an [authentication type](authentication-type/) for REST APIs by using the protocols available on Appsmith.
+You can define an [authentication type](/core-concepts/connecting-to-data-sources/authentication/authentication-type) for REST APIs by using the protocols available on Appsmith.
 
 ## Security
 
-Appsmith safely encrypts all your authentication credentials and stores them securely. Appsmith also does not store any data returned from your data sources and acts only as a proxy layer to orchestrate the API / Query calls. Since Appsmith is an open-source framework, you can [deploy it on-premise](../../../getting-started/setup/), and audit it to ensure none of your data leaves your VPC.
+Appsmith safely encrypts all your authentication credentials and stores them securely. Appsmith also does not store any data returned from your data sources and acts only as a proxy layer to orchestrate the API / Query calls. Since Appsmith is an open-source framework, you can [deploy it on-premise](/getting-started/setup/), and audit it to ensure none of your data leaves your VPC.
```

Comment:
```suggestion
Appsmith safely encrypts all your authentication credentials and stores them securely. Appsmith also does not store any data returned from your data sources and acts only as a proxy layer to orchestrate the API / Query calls. Since Appsmith is an open-source framework, you can [deploy it on-premise](/getting-started/setup), and audit it to ensure none of your data leaves your VPC.
```